### DISCUSSION TITLE
VTX Issues On Inav

### DISCUSSION THREAD

**Discussion by koklimabc** [iNavFlight/inav#8879] [Q&A] [answered]
VTX is not clear and flip upside Down (Sometimes turn up to have black jittering frame) as compared to BetaFlight Won't have this issues.

Does the INAV always required hardware compatibility. Why Don't have VTX Table to setup?

My using SpeedyBee F405V3 connect to AKK FX3 Ultimate (VTX).

**Comment by MrD-RC** [comment, marked as answer]
The video image not being clear or being upside down has nothing to do with VTx Tables. This is most likely down to your camera. Upside down can be fixed on the camera's firmware in most cases. The image clarity will be down to the focus of the lens. The black jittering frames could be that you are on the wrong channel.

VTx tables are not needed. The VTx will dictate which frequencies it can use. All you need to do is set the correct band and channel number.

**Reply by koklimabc** [reply]
Tq, It is fixed.
